src/pages/MapEditorPage/hooks/useCanvasInteractions.js

```
@@ -0,0 +1,449 @@
+    setIsAddingZone,
+    draftZoneVertices,
+    setDraftZoneVertices,
+    setZoneDraftError,
+    editedNodes,
+    editedEdges,
+    nodeShapes,
+    zoneSnapPoints,
+    updateZoneVertices,
+    updateNodePosition,
+    updateSelectedEntityPatch,
+    updateSelectedPoiAttrsPatch,
```

ESLint

[error] 'updateSelectedPoiAttrsPatch' is defined but never used.

(no-unused-vars)